trestle/utils/load_distributed.py

```diff
@@ -56,8 +55,8 @@ def _load_dict(abs_path: Path, abs_trestle_root: Path) -> Tuple[Type[OscalBaseMo
 
 
 def load_distributed(
-    file_path: Path,
-    trestle_root: Path,
+    abs_path: Path,
```

Comment:
I'm not clear what the codecov msg is referring to, but if _load_dict is calling load_distrib and it isn't being tested that's a concern.

Reply:
This is not unexpected. Actually it should have been there for a while now. The changes in OSCAL reduce the use of DICT structures. I'm not sure we are ready to remove this yet, however, if we can it may simplify code in a number of modules.